en/docs/setup/migrating-userstore-managers.md

```diff
@@ -42,13 +42,14 @@ change the `user_id_attribute` to the value of the SCIM ID, in the `<IS_HOME>/re
 
 ## Configurations
 
+Apply the following configurations to the `migration-config.yaml` file:
+
 | Property Name | Description |
 | ------------- | ----------- |
 | migrateAll | Migrate all the tenants and all the user store domains in it. |
 | reportPath | Absolute path for the dry report. This is required in the dry run mode. |
 
-
-The following configurations are only needed to migrate a few tenants. The format should be similar to the one mentioned below. `Tenant-param` is a place holder name to represent each tenant uniquely, e.g., tenant1.
```

Comment:
@ashendes I removed these configs as I am not sure they make much sense. According to [1], the purpose of the user ID migrator is not really this.

[1] https://github.com/wso2-enterprise/wso2-iam-internal/issues/73